src/features/embalm/stepContent/hooks/useRecoverPublicKey.ts

```diff
@@ -99,14 +99,14 @@ export function useRecoverPublicKey() {
 
           return;
         }
-
+        console.log(`retrieving public key for address ${address}`);
```

Comment:
If we will be using console logs in development lets just not forget to override them for production.

Reply:
there is a basic `log` function we could start using, and purposefully gate the logs to environments we define. Will probably be useful to have these logs on testnet webapp, but can remove them for mainnet.

Reply:
We can continue to use `console.log`, just override the `console.log` function based on the environment: 
```
if (process.env.NODE_ENV === 'production') {
  console.log = () => {}
}
```

Reply:
Good call! I swapped this file over to Seth's logger and updated it to stay silent on prod. We'll have a fair amount of `console.log`s to update at some point but I didn't bother with them here.